frb_dart/lib/src/helpers.dart

```diff
@@ -6,6 +6,8 @@ import 'package:flutter_rust_bridge/src/platform_independent.dart';
 import 'package:meta/meta.dart';
 export 'ffi.dart';
 
+const bool kIsWeb = identical(0, 0.0);
```

Comment:
uber-nit: would be great to mention whether this hack copies from

Reply:
resolved along in c4d1ef70954c9e86708fac67a64c6ac19fff6929